content/ai/concepts/generative-ai/concepts/images/images.md

```diff
@@ -0,0 +1,31 @@
+---
+Title: 'Images'
+Description: GenAI can generate new images from existing text prompts and images. Because of a random seed, the images generated are unique creations. GenAI uses diffusion models to create these new unique images.
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
```

Comment:
```suggestion
```
Not on the [subjects list](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).